apps/web/components/AppListCard.tsx

```diff
@@ -85,6 +88,21 @@ export default function AppListCard(props: AppListCardProps) {
             </div>
           )}
         </div>
+        {credentialOwner && (
+          <div>
+            <Badge variant="gray">
+              <div className="flex items-center">
+                <Avatar
+                  className="mr-2"
+                  alt={credentialOwner.name}
+                  size="sm"
```

Comment:
NIT: I think 24px x 24px here might be a bit big 
```suggestion
                  size="xs"
```